benchmarks/benchmarks/stats.py

```diff
@@ -228,3 +228,40 @@ def setup(self, dist_name, loc_fixed, scale_fixed, shape1_fixed,
     def time_fit(self, dist_name, loc_fixed, scale_fixed, shape1_fixed,
                  shape2_fixed, shape3_fixed):
         self.distn.fit(self.data, **self.fixed)
+
+
+class KolmogorovSmirnov(Benchmark):
+    param_names = ['alternative', 'mode', 'size']
+    # No auto since it defaults to exact for 20 samples
+    params = [
+        ['two-sided', 'less', 'greater'],
+        ['exact', 'approx', 'asymp'],
+        [19,20,21]
+    ]
+
+    def setup(self, alternative, mode, size):
+        np.random.seed(12345678)
+        a = stats.norm.rvs(size=20)
+        self.a = a
+
+    def time_ks(self, alternative,mode, size):
+        statistic, pvalue = stats.kstest(self.a,'norm', alternative=alternative,mode=mode,N=size)
```

Comment:
minor: missing a few spaces after commas, plus line is a bit long. can you fix that and put this on two lines or remove `statistic, pvalue =` (don't need to assign values here)?